### PR TITLE
Added Bengali Translation

### PR DESCRIPTION
**Pull request by rabbihossain**
Tried to add Bengali translation.

### CODE REVIEW

File: res/values-bn-rBD/strings.xml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <string name="update_later">পরে আপডেট করুন</string>
+
+    <string name="location_not_found">অবস্থান নির্নয় করা হয়নি</string>
+
+    <string name="gps_alert_msg">জিপিএস বন্ধ আছে। আপনি কি সেটিংস থেকে তা চালু করতে চান?</string>
+    <string name="yes">হ্যা</string>
+    <string name="no">না</string>
+
+    <string name="about_version">সংস্করন %1$s</string>
+    <string name="about_mapbox">ম্যাপিং প্রোভাইডার সম্পর্কে আরও জানুনż/string>
```
Comment (tamcap) [inline res/values-bn-rBD/strings.xml]: You need to replace all instances of ż with < or it won't compile correctly. (1/2)

File: res/values-bn-rBD/strings.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    <string name="about_more">আরওż/string>
```
Comment (tamcap) [inline res/values-bn-rBD/strings.xml]: And here. (2/2)

**Comment by cpeterso** [comment]
@rabbihossain: Here are the error messages (from the Travis-CI build) that @tamcap is referring to:

```
type "string" must be terminated by the matching end-tag "</string>".
MozStumbler/res/values-bn-rBD/strings.xml:55:2: Error: The element type "string" must be terminated by the matching end-tag "</string>".
```

**Comment by cpeterso** [comment]
Also, when you've fixed and committed your value.xml file, can you please use `git rebase` to squash your branch's commits into a single commit? Here are instructions for squashing git commits: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by rabbihossain** [comment]
@cpeterso 
I have fixed the issues and added some more translation in strings.xml . But I didn't understand how to do `git rebase` . As I am noob and I don't have git client installed on my pc.

**Comment by cpeterso** [comment]
Thanks for your help, @rabbihossain!

**Pull request merged**